engine-standalone-storage/src/sync/mod.rs

```diff
@@ -291,10 +299,15 @@ pub fn parse_transaction_kind(
 /// Note: this function does not automatically commit transaction messages to the storage.
 /// If you want the transaction diff committed then you must call the `commit` method on
 /// the outcome of this function.
-pub fn consume_message<M: ModExpAlgorithm + 'static>(
+pub fn consume_message<M: ModExpAlgorithm + 'static, R>(
```

Comment:
It feels somewhat convoluted to me that we are changing the standalone running to work in this way. The original point of the standalone runner is that it provided us a pathway to executing Engine transactions without Wasm.

Now we want to execute transactions in an alternate Wasm runner rather than native. But then I think the standalone runner interface no longer makes sense. We are consuming structured data only to convert it into strings and bytes to pass into the Wasm runner. It seems like it would be more direct to have an alternate interface for Borealis Engine where it can pass Near `FunctionCall` actions as they are instead of processing them into structured data (`Message`) only to then turn it back into the unstructured form of the original `FunctionCall`.